certbot/main.py

```diff
@@ -690,6 +690,7 @@ def certonly(config, plugins):
 def renew(config, unused_plugins):
     """Renew previously-obtained certificates."""
     try:
+        config.renewing = "yes"
```

Comment:
This method is called iff the `renew` verb is used, so this line is currently redundant and `config.renewing` can be removed entirely, and only `config.verb` inspected.